packages/features/apps/components/AppCard.tsx

```
@@ -114,14 +115,14 @@ export function AppCard({ app, credentials, searchText, userAdminTeams }: AppCar
           </div> */}
       <p
         className="text-default mt-2 flex-grow text-sm"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: markdownToSafeHTML(app.description) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)